atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/TargetedSweeper.java

```diff
@@ -163,11 +172,16 @@ public void sweepNextBatch(ShardAndStrategy shardStrategy) {
     }
 
     @Override
-    public void close() throws Exception {
+    public void close() {
         conservativeScheduler.close();
         thoroughScheduler.close();
```

Comment:
random thought: if the conservative scheduler throws a runtime exception then the thorough scheduler is never closed. Not a regression, and non-blocking.

Reply:
Will fix in a separate PR